java-client/session/src/main/java/io/deephaven/client/impl/AggregationBuilder.java

```diff
@@ -25,7 +25,7 @@
 
 import static java.util.Collections.singletonList;
 
-class AggregationBuilder implements io.deephaven.api.agg.Aggregation.Visitor {
+public class AggregationBuilder implements io.deephaven.api.agg.Aggregation.Visitor {
```

Comment:
The method on 30 is public, but the class wasn't.  I think this is reasonable to allow other services like Pivots to convert an engine aggregation to a gRPC aggregation.  The inverse was already possible.

It probably does mean a Javadoc comment is in order if there is no objection.